CMakeLists.txt

```
@@ -0,0 +1,16 @@
+project(asio-rawsockets-example)
+cmake_minimum_required(VERSION 3.2)
+
+add_compile_options(-std=c++11)
+
+find_package(Boost REQUIRED system)
+
+INCLUDE_DIRECTORIES(${Boost_INCLUDE_DIR})
+
+# Building server
+add_executable(example
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/example.cpp
+)
+
+target_link_libraries(example LINK_PUBLIC ${Boost_LIBRARIES}     
```

I'd suggest to use `PRIVATE` instead since we don't have any other targets right now.

```suggestion
target_link_libraries(example PRIVATE ${Boost_LIBRARIES}     
```

---

Agreed